app/models/widget/card.rb

```diff
@@ -1,8 +1,15 @@
 class Widget::Card < ActiveRecord::Base
   include Imageable
 
+  # table_name must be set before calls to 'translates'
   self.table_name = "widget_cards"
 
+  translates :label,       touch: true
+  translates :title,       touch: true
+  translates :description, touch: true
+  translates :link_text,   touch: true
+  globalize_accessors locales: [:en, :es, :fr, :nl, :val, :pt_br]
```

Comment:
Hi, @mlovic. Since the languages seem to always be `[:en, :es, :fr, :nl, :val, :pt_br]` on this project and `[:en, :es, :fr, :nl, :pt_br]` in CONSUL's main repo, I was wondering if it makes sense to have them in a configuration file. That way we remove duplication, and backporting these PRs will be easier. What do you think?

Reply:
Good idea. It looks like those are the same locales that are [registered with I18n](https://github.com/AyuntamientoMadrid/consul/blob/master/config/application.rb#L23), except that 'pt-BR' needs to be underscored ([apparently](https://github.com/AyuntamientoMadrid/consul/pull/1435) this gem does not handle uppercase or `-`).

How about:

```ruby
  globalize_accessors locales: I18n.available_locales.map(&:to_s).map(&:underscore)
```

Reply:
I hadn't realized they were already defined. Great catch!

Reply:
Re: `pt-BR`, @iagirre documented this case very well on consul#2619

Reply:
Thanks @aitbw :+1: . I tried to link to that from my last message.